### PR TITLE
🚧 Add api for stats

### PR DESCRIPTION
**Pull request by Ic3Sandy**

### CODE REVIEW

File: packages/bitcore-node/src/modules/firocoin/api/stats.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if (cache) {
+      return res.json(cache);
+    }
+    const totalTxns = await TransactionStorage.collection.countDocuments();
+    const totalBlocks = await BitcoinBlockStorage.collection.countDocuments();
+    const walletAddress = await WalletAddressStorage.collection.countDocuments();
+    let blockTime = 0;
+    if (totalBlocks > 1) {
+      const blocks = await BitcoinBlockStorage.collection.find({}).sort({ _id: -1 }).limit(2).toArray();
+      blockTime = Math.abs(+blocks[0].time - +blocks[1].time) / 1000; // sec
```
Comment (thebevrishot) [inline packages/bitcore-node/src/modules/firocoin/api/stats.ts]: This should be average that could be calculate from (last_block_time - 1st_block_time)/(last_block_num - 1)

**Review by thebevrishot — CHANGES_REQUESTED** [review]

File: packages/bitcore-node/src/modules/firocoin/api/stats.ts (hunk 1 above)
Comment (Ic3Sandy) [reply packages/bitcore-node/src/modules/firocoin/api/stats.ts]: ok kub

File: packages/bitcore-node/src/modules/firocoin/api/gashistory.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    res.json(result);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send(resMessage((<any>err).message));
+  }
+});
+
+module.exports = {
+  router,
+  path: '/gashistory',
```
Comment (thebevrishot) [inline packages/bitcore-node/src/modules/firocoin/api/gashistory.ts]: we could put it to /stats/gashistory. We could do it with txnshistory also

Comment (Ic3Sandy) [reply packages/bitcore-node/src/modules/firocoin/api/gashistory.ts]: roger kub

**Review by thebevrishot — APPROVED** [review]

**Pull request merged**